tests/testthat/test-autoplot.summary.seroincidence.by.R

```diff
@@ -51,3 +51,133 @@ test_that(
 
   }
 )
+
+test_that(
+  desc = "barplot results are consistent",
+  code = {
+
+
+    library(dplyr)
+    library(ggplot2)
+
+    xs_data <-
+      sees_pop_data_pk_100
+
+    curve <-
+      typhoid_curves_nostrat_100 |>
+      filter(antigen_iso %in% c("HlyE_IgA", "HlyE_IgG"))
+
+    noise <-
+      example_noise_params_pk
+
+    est2 <- estimate_scr_by(
+      strata = c("catchment", "ageCat"),
+      pop_data = xs_data,
+      curve_params = curve,
+      noise_params = noise,
+      curve_strata_varnames = NULL,
+      noise_strata_varnames = NULL,
+      antigen_isos = c("HlyE_IgG", "HlyE_IgA"),
+      num_cores = 2 # Allow for parallel processing to decrease run time
+    )
+
+    est2sum <- summary(est2)
+
+    plot1 <- autoplot(est2sum,
+                      yvar = "ageCat",
+                      type = "bar",
+                      dodge_width = 0.1,
+                      color_var = "catchment",
+                      CI = TRUE)
+
+    plot1 |> vdiffr::expect_doppelganger(title = "strat-est-barplot")
+
+    plot2 <- autoplot(est2sum,
+                      yvar = "ageCat",
+                      type = "bar",
+                      CI = TRUE,
+                      dodge_width = 0.1,
+                      color_var = "catchment")
+
+    plot2 |> vdiffr::expect_doppelganger(title = "strat-est-barplot")
+
+
+  }
+)
+
+test_that("Invalid plot `type` specified. Please choose either 'scatter' or 'bar'.",
+  expect_snapshot(error = TRUE, {
```

Comment:
```suggestion
{
```
the `expect_snapshot()` call goes inside the `{}` of `testthat(desc, {<CODE-HERE>})`; see below